src/components/Followers/Followers.js

```diff
@@ -66,13 +85,22 @@ const Followers = () => {
   }
 
   return (
-    <div>
-    <h2>Fixed Followers</h2>
-      <h3>Followers for Current user ID{value}</h3>
-      {!!data && renderFollowers()}
-      <h3>Following</h3>
-      {!!GetFollowingInfo.data && renderFollowing()}
-    </div>
+    <section className="followers-containers">
+      <div>
+        <h2>Fixed</h2>
```

Comment:
Maybe we can turn this into it's own component so that we don't have code smell and redundant code